src/main.ts

```diff
@@ -57,10 +58,48 @@ const mainExecution = (program) => {
     /**
      * Push section
      */
-    const _push = (forceFrom: string|null, callback: ErrOnlyCallback): void => {
+    const _push = (forceFrom: string | null, callback: ErrOnlyCallback): void => {
         push(forceFrom || program.push, program.source, program.deploy, callback);
     };
 
+    /**
+     * Push multiple packages
+     */
+    const _multiplePush = (callback: ErrOnlyCallback): void => {
+        const fromPath = program.push;
```

Comment:
Ho... I read this comment too fast. It's true `fromPath` is ugly and don't describe well what the variable is. I will change that.